src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/HomeLowRiskCellConfigurator.swift

```diff
@@ -70,7 +70,8 @@ final class HomeLowRiskCellConfigurator: HomeRiskLevelCellConfigurator {
 			let numberDaysString = String(numberDays)
 			let totalDaysString = String(totalDays)
 			let saveDays = String(format: AppStrings.Home.riskCardLowSaveDaysItemTitle, numberDaysString, totalDaysString)
-			let item2 = HomeRiskImageItemViewConfigurator(title: saveDays, titleColor: titleColor, iconImageName: "Icons_TracingCircleFull - Dark", iconTintColor: titleColor, color: color, separatorColor: separatorColor)
+			let progressImage: String = numberDays >= totalDays ? "Icons_TracingCircleFull - Dark" : "Icons_TracingCircle-Dark_Step \(numberDays)"
+			let item2 = HomeRiskImageItemViewConfigurator(title: saveDays, titleColor: titleColor, iconImageName: progressImage, iconTintColor: titleColor, color: color, separatorColor: separatorColor)
```

Comment:
We could re-use the circular progress indicator @haosap wrote - right? :)

Reply:
I wouldn't suggest that, as it is not aligned with the iconography. Also it is more difficult to to a custom view at this point (same goes for risk detail) than sticking with icons.